src/app/components/client/ExposureCard.module.scss

```diff
@@ -84,25 +95,46 @@
         list-style: none;
```

Comment:
Above on L34 we changed this to `list-style-type: none;`
Not sure if that needs to change here as well or why it changed above.